java-strings/src/test/java/com/baeldung/stringduplicates/RemoveDuplicateFromStringUnitTest.java

```diff
@@ -0,0 +1,72 @@
+package com.baeldung.stringduplicates;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class RemoveDuplicateFromStringUnitTest {
+
+    private final static String STR1 = "racecar";
+    private final static String STR2 = "J2ee programming";
+    private RemoveDuplicateFromString removeDuplicateFromString;
+
+    @Before
+    public void executedBeforeEach() {
+        removeDuplicateFromString = new RemoveDuplicateFromString();
+    }
+
+
+    @Test
+    public void whenUsingCharArray_DuplicatesShouldBeRemovedWithoutKeepingStringOrder() {
+        String str1 = removeDuplicateFromString.removeDuplicatesUsingCharArray(STR1);
+        String str2 = removeDuplicateFromString.removeDuplicatesUsingCharArray(STR2);
+        Assert.assertEquals("", "");
```

Comment:
What is the point of these empty string asserts?

Reply:
I was trying to test for empty strings but I forget to add the below line
 String strEmpty = removeDuplicateFromString.removeDuplicatesUsingSorting(STR_EMPTY);
will be fixed

Reply:
Done